go.mod

```diff
@@ -16,11 +16,11 @@ require (
 	github.com/jedib0t/go-pretty/v6 v6.4.4
 	github.com/joshdk/go-junit v0.0.0-20210226021600-6145f504ca0d
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/newrelic/newrelic-client-go/v2 v2.19.6
+	github.com/newrelic/newrelic-client-go/v2 v2.20.1-0.20230825151912-bb7c5324adab
```

Comment:
Note to the reviewer: The import will be updated once the Client is released. The current commit is being used tentatively for testing purposes.